src/ir/value.h

```diff
@@ -124,8 +126,8 @@ class Any {};
 // A class that represents a data value.
 class Value {
  public:
-  Value(value::BlockArgument arg): value_(std::move(arg)) {}
-  Value(value::BlockResult arg): value_(std::move(arg)) {}
+  Value(value::BlockArgument arg) : value_(std::move(arg)) {}
+  Value(value::BlockResult arg) : value_(std::move(arg)) {}
```

Comment:
This change also seems a little off-theme for this PR.

Reply:
reverted this.